internal/hcsoci/hcsdoc_wcow.go

```diff
@@ -416,10 +416,23 @@ func createWindowsContainerDocument(ctx context.Context, coi *createOptionsInter
 	}
 
 	if dumpPath != "" {
+		// Create subdirectory for this container at the specified dumpPath location when
+		// CreateContainerSubdirectoryForProcessDump annotation is set.
+		// When a fileshare from the host is mounted to the specified dumpPath, this behavior will
+		// help identify dumps coming from differnet containers in the pod.
+		if createContainerSubdirectory, ok := coi.Spec.Annotations[annotations.CreateContainerSubdirectoryForProcessDump]; ok {
```

Comment:
`"internal\oci\annotations.go".ParseAnnotationsBool`
or at least add a `strings.ToLower` to make it case insensitive